client/src/styles/global.scss

```diff
@@ -1,2 +1,31 @@
-@import "~@trussworks/react-uswds/lib/uswds.css";
 @import "~@trussworks/react-uswds/lib/index.css";
+@import "~@trussworks/react-uswds/lib/uswds.css";
+
+.j40FloatLogoLeft {
+  width: 100%;
+  padding-left: 50%;
+}
+
+.j40Footer {
+  font-family: "sans-serif";
```

Comment:
I wonder if we can keep this as default and set this at the [theme](https://designsystem.digital.gov/documentation/developers/#sass-and-theme-settings) level if need-be?

Reply:
Good call. I was experimenting with setting the font and this should go away.